.github/workflows/coverage.yml

```
@@ -4,6 +4,9 @@ on:
   workflow_dispatch: # Trigger from UI for selected branch
   schedule:
     - cron: "0 0 * * *"
+  push:
+    branches:
+      - *coverage* # Trigger on branches ending with 'coverage'
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 9: did not find expected alphabetic or numeric character

(syntax-check)